packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -228,7 +228,7 @@ export const useIsEmbed = (embedSsr?: boolean) => {
         "Looks like you have iframed cal.com but not using Embed Snippet. Directly using an iframe isn't recommended."
       );
     }
-    setIsEmbed(window.isEmbed());
+    setIsEmbed(window?.isEmbed?.() || false);
```

Comment:
TIL `?.()` is the way to call an undefinable function.

Reply:
Optional chaining for functions 🔥